src/SizeBench.TestInfrastructure/STATestClassAttribute.cs

```diff
@@ -1,20 +0,0 @@
-﻿namespace SizeBench.TestInfrastructure;
-
-// Copied from https://github.com/microsoft/PowerToys/blob/master/src/modules/previewpane/STATestClassAttribute.cs
-public sealed class STATestClassAttribute : TestClassAttribute

```

Comment:
>STATestClassAttribute [](http://example.com/codeflow?start=20&length=21)

MSTest now has a built-in `STATestClass` and `STATestMethod` - hooray!  Now SizeBench doesn't need its own.